### PR TITLE
update release-tasklist

### PR DESCRIPTION
**Pull request by tjhei**

### CODE REVIEW

File: doc/release-tasklist [hunk 1]
```diff
@@ -73,6 +84,7 @@
       gpg --detach-sign --armor manual-$TAG.pdf
       sha1sum aspect-$TAG.tar.gz manual-$TAG.pdf >sha1sum-$TAG.txt
   . create a release on github, upload tar.gz and manual-$VER.pdf
+  . readme / release notes: add zenodo DOI button
```
Comment (gassmoeller) [inline doc/release-tasklist]: Can you elaborate on this a bit more? I would not know where or how to add a DOI button.

File: doc/release-tasklist [hunk 2]
```diff
@@ -57,13 +57,24 @@
       cd doc && ./bump_version.sh $TAG && cd .. && \
       git commit -m "version $TAG" && \
       git tag -s v$TAG -m "version $TAG"
-  . create a tar file:
+  . create a tar file: TODO: remove manual.pdf
```
Comment (gassmoeller) [inline doc/release-tasklist]: What about this TODO?

Comment (tjhei) [reply doc/release-tasklist]: I thought I don't spend the time right now to change and test the commands to not include the manual.pdf in the tarball but do this next time. Do you want me to update the bash commands now?

File: doc/release-tasklist (hunk 1 above)
Comment (tjhei) [reply doc/release-tasklist]: yeah. I think this is something we should discuss before we put this into effect. Basically, @ljhwang said we could do something like this: https://github.com/geodynamics/pylith/releases/tag/v2.2.1

Comment (gassmoeller) [reply doc/release-tasklist]: Seems nice. Are there arguments against this?

File: doc/release-tasklist (hunk 2 above)
Comment (gassmoeller) [reply doc/release-tasklist]: That would be great, yes.

**Comment by tjhei** [comment]
updated. Okay like this?

**Pull request merged**